### ISSUE TITLE
[CI] IndexRecoveryIT#testRerouteRecovery cannot find files

### ISSUE THREAD

**Issue by talevy** [elastic/elasticsearch#26131]
There is a strange issue in `testRerouteRecovery` where certain expected metadata files cannot be found in the test instances.

CI run: https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+master+aggressive-opts/546/console

reproduce:

```
gradle :core:integTest \
  -Dtests.seed=B83A63FDE9F84795 \
  -Dtests.class=org.elasticsearch.indices.recovery.IndexRecoveryIT \
  -Dtests.method="testRerouteRecovery" \
  -Dtests.security.manager=true \
  -Dtests.jvm.argline="-XX:+AggressiveOpts" \
  -Dtests.locale=ar-BH \
  -Dtests.timezone=America/Moncton
```

snippets:

```
 1> 	at java.lang.Thread.run(Thread.java:748) ~[?:1.8.0_141]
  1> Caused by: org.elasticsearch.transport.RemoteTransportException: [node_t1][127.0.0.1:54893][internal:index/shard/recovery/start_recovery]
  1> Caused by: org.elasticsearch.transport.ReceiveTimeoutTransportException: [node_t2][127.0.0.1:9662][internal:index/shard/recovery/finalize] request_id [202] timed out after [2000ms]
```

```
  1> [2017-08-09T09:55:59,153][TRACE][o.e.c.s.MasterService$AckCountDownListener] ack received from node [{node_t1}{IHnJS1UwSCuZ3p-MZ4YzUQ}{1ukoSfBNTfGoxPzTPa8VRg}{127.0.0.1}{127.0.0.1:9601}], cluster_state update (version: 12)
  1> [2017-08-09T09:55:59,156][DEBUG][o.e.i.s.TransportNodesListShardStoreMetaData] [node_t0] /var/lib/jenkins/workspace/elastic+elasticsearch+master+aggressive-opts/core/build/testrun/integTest/J2/temp/org.elasticsearch.indices.recovery.IndexRecoveryIT_B83A63FDE9F84795-001/tempDir-004/data/nodes/0/indices/wZiPN8daSeOhZeePMqpm4A/0/_state/state-1.st: failed to read [state-], ignoring...
  1> java.nio.file.NoSuchFileException: /var/lib/jenkins/workspace/elastic+elasticsearch+master+aggressive-opts/core/build/testrun/integTest/J2/temp/org.elasticsearch.indices.recovery.IndexRecoveryIT_B83A63FDE9F84795-001/tempDir-004/data/nodes/0/indices/wZiPN8daSeOhZeePMqpm4A/0/_state/state-1.st
  1> 	at sun.nio.fs.UnixException.translateToIOException(UnixException.java:86) ~[?:?]
  1> 	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:102) ~[?:?]
  1> 	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:107) ~[?:?]
  1> 	at sun.nio.fs.UnixFileSystemProvider.newByteChannel(UnixFileSystemProvider.java:214) ~[?:?]
  1> 	at org.apache.lucene.mockfile.FilterFileSystemProvider.newByteChannel(FilterFileSystemProvider.java:212) ~[lucene-test-framework-7.0.0-snapshot-a128fcb.jar:7.0.0-snapshot-a128fcb a128fcb8444271d73f36744018b5261b3bff0606 - jpountz - 2017-08-08 08:38:18]
  1> 	at org.apache.lucene.mockfile.FilterFileSystemProvider.newByteChannel(FilterFileSystemProvider.java:212) ~[lucene-test-framework-7.0.0-snapshot-a128fcb.jar:7.0.0-snapshot-a128fcb a128fcb8444271d73f36744018b5261b3bff0606 - jpountz - 2017-08-08 08:38:18]
  1> 	at org.apache.lucene.mockfile.FilterFileSystemProvider.newByteChannel(FilterFileSystemProvider.java:212) ~[lucene-test-framework-7.0.0-snapshot-a128fcb.jar:7.0.0-snapshot-a128fcb a128fcb8444271d73f36744018b5261b3bff0606 - jpountz - 2017-08-08 08:38:18]
  1> 	at org.apache.lucene.mockfile.HandleTrackingFS.newByteChannel(HandleTrackingFS.java:240) ~[lucene-test-framework-7.0.0-snapshot-a128fcb.jar:7.0.0-snapshot-a128fcb a128fcb8444271d73f36744018b5261b3bff0606 - jpountz - 2017-08-08 08:38:18]
  1> 	at org.apache.lucene.mockfile.FilterFileSystemProvider.newByteChannel(FilterFileSystemProvider.java:212) ~[lucene-test-framework-7.0.0-snapshot-a128fcb.jar:7.0.0-snapshot-a128fcb a128fcb8444271d73f36744018b5261b3bff0606 - jpountz - 2017-08-08 08:38:18]
  1> 	at org.apache.lucene.mockfile.HandleTrackingFS.newByteChannel(HandleTrackingFS.java:240) ~[lucene-test-framework-7.0.0-snapshot-a128fcb.jar:7.0.0-snapshot-a128fcb a128fcb8444271d73f36744018b5261b3bff0606 - jpountz - 2017-08-08 08:38:18]
  1> 	at java.nio.file.Files.newByteChannel(Files.java:361) ~[?:1.8.0_141]
  1> 	at java.nio.file.Files.newByteChannel(Files.java:407) ~[?:1.8.0_141]
  1> 	at org.apache.lucene.store.SimpleFSDirectory.openInput(SimpleFSDirectory.java:77) ~[lucene-core-7.0.0-snapshot-a128fcb.jar:7.0.0-snapshot-a128fcb a128fcb8444271d73f36744018b5261b3bff0606 - jpountz - 2017-08-08 08:38:17]
  1> 	at org.elasticsearch.gateway.MetaDataStateFormat.read(MetaDataStateFormat.java:187) ~[main/:?]
  1> 	at org.elasticsearch.gateway.MetaDataStateFormat.loadLatestState(MetaDataStateFormat.java:322) ~[main/:?]
  1> 	at org.elasticsearch.index.shard.ShardPath.loadShardPath(ShardPath.java:118) ~[main/:?]
  1> 	at org.elasticsearch.indices.store.TransportNodesListShardStoreMetaData.listStoreMetaData(TransportNodesListShardStoreMetaData.java:142) ~[main/:?]
  1> 	at org.elasticsearch.indices.store.TransportNodesListShardStoreMetaData.nodeOperation(TransportNodesListShardStoreMetaData.java:109) ~[main/:?]
  1> 	at org.elasticsearch.indices.store.TransportNodesListShardStoreMetaData.nodeOperation(TransportNodesListShardStoreMetaData.java:61) ~[main/:?]
  1> 	at org.elasticsearch.action.support.nodes.TransportNodesAction.nodeOperation(TransportNodesAction.java:140) ~[main/:?]
  1> 	at org.elasticsearch.action.support.nodes.TransportNodesAction$NodeTransportHandler.messageReceived(TransportNodesAction.java:262) ~[main/:?]
  1> 	at org.elasticsearch.action.support.nodes.TransportNodesAction$NodeTransportHandler.messageReceived(TransportNodesAction.java:258) ~[main/:?]
  1> 	at org.elasticsearch.transport.RequestHandlerRegistry.processMessageReceived(RequestHandlerRegistry.java:66) ~[main/:?]
  1> 	at org.elasticsearch.transport.TransportService$7.doRun(TransportService.java:650) ~[main/:?]
  1> 	at org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:638) ~[main/:?]
  1> 	at org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:37) ~[main/:?]
  1> 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[?:1.8.0_141]
  1> 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[?:1.8.0_141]
  1> 	at java.lang.Thread.run(Thread.java:748) ~[?:1.8.0_141]
  1> [2017-08-09T09:55:59,158][DEBUG][o.e.i.s.TransportNodesListShardStoreMetaData] [node_t0] failed to execute on node [7O4zoDLUQzG7XYe1WEYPmg]
  1> org.elasticsearch.transport.RemoteTransportException: [node_t0][127.0.0.1:9600][internal:cluster/nodes/indices/shard/store[n]]
  1> Caused by: org.elasticsearch.ElasticsearchException: java.io.IOException: failed to read [id:1, legacy:false, file:/var/lib/jenkins/workspace/elastic+elasticsearch+master+aggressive-opts/core/build/testrun/integTest/J2/temp/org.elasticsearch.indices.recovery.IndexRecoveryIT_B83A63FDE9F84795-001/tempDir-004/data/nodes/0/indices/wZiPN8daSeOhZeePMqpm4A/0/_state/state-1.st]
  1> 	at org.elasticsearch.ExceptionsHelper.maybeThrowRuntimeAndSuppress(ExceptionsHelper.java:150) ~[main/:?]
  1> 	at org.elasticsearch.gateway.MetaDataStateFormat.loadLatestState(MetaDataStateFormat.java:334) ~[main/:?]
  1> 	at org.elasticsearch.index.shard.ShardPath.loadShardPath(ShardPath.java:118) ~[main/:?]
  1> 	at org.elasticsearch.indices.store.TransportNodesListShardStoreMetaData.listStoreMetaData(TransportNodesListShardStoreMetaData.java:142) ~[main/:?]
  1> 	at org.elasticsearch.indices.store.TransportNodesListShardStoreMetaData.nodeOperation(TransportNodesListShardStoreMetaData.java:109) ~[main/:?]
  1> 	at org.elasticsearch.indices.store.TransportNodesListShardStoreMetaData.nodeOperation(TransportNodesListShardStoreMetaData.java:61) ~[main/:?]
  1> 	at org.elasticsearch.action.support.nodes.TransportNodesAction.nodeOperation(TransportNodesAction.java:140) ~[main/:?]
  1> 	at org.elasticsearch.action.support.nodes.TransportNodesAction$NodeTransportHandler.messageReceived(TransportNodesAction.java:262) ~[main/:?]
  1> 	at org.elasticsearch.action.support.nodes.TransportNodesAction$NodeTransportHandler.messageReceived(TransportNodesAction.java:258) ~[main/:?]
  1> 	at org.elasticsearch.transport.RequestHandlerRegistry.processMessageReceived(RequestHandlerRegistry.java:66) ~[main/:?]
  1> 	at org.elasticsearch.transport.TransportService$7.doRun(TransportService.java:650) ~[main/:?]
  1> 	at org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:638) ~[main/:?]
  1> 	at org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:37) ~[main/:?]
  1> 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[?:1.8.0_141]
  1> 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[?:1.8.0_141]
  1> 	at java.lang.Thread.run(Thread.java:748) ~[?:1.8.0_141]
  1> Caused by: java.io.IOException: failed to read [id:1, legacy:false, file:/var/lib/jenkins/workspace/elastic+elasticsearch+master+aggressive-opts/core/build/testrun/integTest/J2/temp/org.elasticsearch.indices.recovery.IndexRecoveryIT_B83A63FDE9F84795-001/tempDir-004/data/nodes/0/indices/wZiPN8daSeOhZeePMqpm4A/0/_state/state-1.st]
  1> 	at org.elasticsearch.gateway.MetaDataStateFormat.loadLatestState(MetaDataStateFormat.java:327) ~[main/:?]
  1> 	at org.elasticsearch.index.shard.ShardPath.loadShardPath(ShardPath.java:118) ~[main/:?]
  1> 	at org.elasticsearch.indices.store.TransportNodesListShardStoreMetaData.listStoreMetaData(TransportNodesListShardStoreMetaData.java:142) ~[main/:?]
  1> 	at org.elasticsearch.indices.store.TransportNodesListShardStoreMetaData.nodeOperation(TransportNodesListShardStoreMetaData.java:109) ~[main/:?]
  1> 	at org.elasticsearch.indices.store.TransportNodesListShardStoreMetaData.nodeOperation(TransportNodesListShardStoreMetaData.java:61) ~[main/:?]
  1> 	at org.elasticsearch.action.support.nodes.TransportNodesAction.nodeOperation(TransportNodesAction.java:140) ~[main/:?]
  1> 	at org.elasticsearch.action.support.nodes.TransportNodesAction$NodeTransportHandler.messageReceived(TransportNodesAction.java:262) ~[main/:?]
  1> 	at org.elasticsearch.action.support.nodes.TransportNodesAction$NodeTransportHandler.messageReceived(TransportNodesAction.java:258) ~[main/:?]
  1> 	at org.elasticsearch.transport.RequestHandlerRegistry.processMessageReceived(RequestHandlerRegistry.java:66) ~[main/:?]
  1> 	at org.elasticsearch.transport.TransportService$7.doRun(TransportService.java:650) ~[main/:?]
  1> 	at org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:638) ~[main/:?]
  1> 	at org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:37) ~[main/:?]
  1> 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[?:1.8.0_141]
  1> 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[?:1.8.0_141]
  1> 	at java.lang.Thread.run(Thread.java:748) ~[?:1.8.0_141]
  1> Caused by: java.nio.file.NoSuchFileException: /var/lib/jenkins/workspace/elastic+elasticsearch+master+aggressive-opts/core/build/testrun/integTest/J2/temp/org.elasticsearch.indices.recovery.IndexRecoveryIT_B83A63FDE9F84795-001/tempDir-004/data/nodes/0/indices/wZiPN8daSeOhZeePMqpm4A/0/_state/state-1.st
  1> 	at sun.nio.fs.UnixException.translateToIOException(UnixException.java:86) ~[?:?]
  1> 	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:102) ~[?:?]
  1> 	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:107) ~[?:?]
  1> 	at sun.nio.fs.UnixFileSystemProvider.newByteChannel(UnixFileSystemProvider.java:214) ~[?:?]
  1> 	at org.apache.lucene.mockfile.FilterFileSystemProvider.newByteChannel(FilterFileSystemProvider.java:212) ~[lucene-test-framework-7.0.0-snapshot-a128fcb.jar:7.0.0-snapshot-a128fcb a128fcb8444271d73f36744018b5261b3bff0606 - jpountz - 2017-08-08 08:38:18]
  1> 	at org.apache.lucene.mockfile.FilterFileSystemProvider.newByteChannel(FilterFileSystemProvider.java:212) ~[lucene-test-framework-7.0.0-snapshot-a128fcb.jar:7.0.0-snapshot-a128fcb a128fcb8444271d73f36744018b5261b3bff0606 - jpountz - 2017-08-08 08:38:18]
  1> 	at org.apache.lucene.mockfile.FilterFileSystemProvider.newByteChannel(FilterFileSystemProvider.java:212) ~[lucene-test-framework-7.0.0-snapshot-a128fcb.jar:7.0.0-snapshot-a128fcb a128fcb8444271d73f36744018b5261b3bff0606 - jpountz - 2017-08-08 08:38:18]
  1> 	at org.apache.lucene.mockfile.HandleTrackingFS.newByteChannel(HandleTrackingFS.java:240) ~[lucene-test-framework-7.0.0-snapshot-a128fcb.jar:7.0.0-snapshot-a128fcb a128fcb8444271d73f36744018b5261b3bff0606 - jpountz - 2017-08-08 08:38:18]
  1> 	at org.apache.lucene.mockfile.FilterFileSystemProvider.newByteChannel(FilterFileSystemProvider.java:212) ~[lucene-test-framework-7.0.0-snapshot-a128fcb.jar:7.0.0-snapshot-a128fcb a128fcb8444271d73f36744018b5261b3bff0606 - jpountz - 2017-08-08 08:38:18]
  1> 	at org.apache.lucene.mockfile.HandleTrackingFS.newByteChannel(HandleTrackingFS.java:240) ~[lucene-test-framework-7.0.0-snapshot-a128fcb.jar:7.0.0-snapshot-a128fcb a128fcb8444271d73f36744018b5261b3bff0606 - jpountz - 2017-08-08 08:38:18]
  1> 	at java.nio.file.Files.newByteChannel(Files.java:361) ~[?:1.8.0_141]
  1> 	at java.nio.file.Files.newByteChannel(Files.java:407) ~[?:1.8.0_141]
  1> 	at org.apache.lucene.store.SimpleFSDirectory.openInput(SimpleFSDirectory.java:77) ~[lucene-core-7.0.0-snapshot-a128fcb.jar:7.0.0-snapshot-a128fcb a128fcb8444271d73f36744018b5261b3bff0606 - jpountz - 2017-08-08 08:38:17]
  1> 	at org.elasticsearch.gateway.MetaDataStateFormat.read(MetaDataStateFormat.java:187) ~[main/:?]
  1> 	at org.elasticsearch.gateway.MetaDataStateFormat.loadLatestState(MetaDataStateFormat.java:322) ~[main/:?]
  1> 	at org.elasticsearch.index.shard.ShardPath.loadShardPath(ShardPath.java:118) ~[main/:?]
  1> 	at org.elasticsearch.indices.store.TransportNodesListShardStoreMetaData.listStoreMetaData(TransportNodesListShardStoreMetaData.java:142) ~[main/:?]
  1> 	at org.elasticsearch.indices.store.TransportNodesListShardStoreMetaData.nodeOperation(TransportNodesListShardStoreMetaData.java:109) ~[main/:?]
  1> 	at org.elasticsearch.indices.store.TransportNodesListShardStoreMetaData.nodeOperation(TransportNodesListShardStoreMetaData.java:61) ~[main/:?]
  1> 	at org.elasticsearch.action.support.nodes.TransportNodesAction.nodeOperation(TransportNodesAction.java:140) ~[main/:?]
  1> 	at org.elasticsearch.action.support.nodes.TransportNodesAction$NodeTransportHandler.messageReceived(TransportNodesAction.java:262) ~[main/:?]
  1> 	at org.elasticsearch.action.support.nodes.TransportNodesAction$NodeTransportHandler.messageReceived(TransportNodesAction.java:258) ~[main/:?]
  1> 	at org.elasticsearch.transport.RequestHandlerRegistry.processMessageReceived(RequestHandlerRegistry.java:66) ~[main/:?]
  1> 	at org.elasticsearch.transport.TransportService$7.doRun(TransportService.java:650) ~[main/:?]
  1> 	at org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:638) ~[main/:?]
  1> 	at org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:37) ~[main/:?]
  1> 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[?:1.8.0_141]
  1> 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[?:1.8.0_141]
  1> 	at java.lang.Thread.run(Thread.java:748) ~[?:1.8.0_141]
  1> [2017-08-09T09:55:59,182][DEBUG][o.e.i.s.TransportNodesListShardStoreMetaData] [node_t1] [test-idx-1][0] loaded store meta data (took [32ms])
  1> [2017-08-09T09:55:59,183][WARN ][o.e.g.GatewayAllocator$InternalReplicaShardAllocator] [node_t0] [test-idx-1][0]: failed to list shard for shard_store on node [7O4zoDLUQzG7XYe1WEYPmg]
  1> org.elasticsearch.action.FailedNodeException: Failed node [7O4zoDLUQzG7XYe1WEYPmg]
  1> 	at 
```

[consoleText.txt](https://github.com/elastic/elasticsearch/files/1213155/consoleText.txt)

**Comment by bleskes** [comment]
This is so old, I don't think it's worth chasing down. closing.

**Comment by vladimirdolzhenko** [comment]
Recent failure of this test: 

Log: https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+6.2+matrix-java-periodic/ES_BUILD_JAVA=java9,ES_RUNTIME_JAVA=java10,nodes=virtual&&linux/58/console

Root cause:
```
2> WARNING: Uncaught exception in thread: Thread[elasticsearch[node_t0][clusterApplierService#updateTask][T#1],5,TGRP-IndexRecoveryIT]
  2> java.lang.AssertionError
  2> 	at __randomizedtesting.SeedInfo.seed([B83A63FDE9F84795]:0)
  2> 	at org.elasticsearch.env.NodeEnvironment.deleteShardDirectoryUnderLock(NodeEnvironment.java:460)
  2> 	at org.elasticsearch.env.NodeEnvironment.deleteShardDirectorySafe(NodeEnvironment.java:400)
  2> 	at org.elasticsearch.indices.IndicesService.deleteShardStore(IndicesService.java:710)
  2> 	at org.elasticsearch.indices.store.IndicesStore$ShardActiveResponseHandler.lambda$allNodesResponded$2(IndicesStore.java:289)
  2> 	at org.elasticsearch.cluster.service.ClusterApplierService.lambda$runOnApplierThread$0(ClusterApplierService.java:301)
  2> 	at org.elasticsearch.cluster.service.ClusterApplierService$UpdateTask.apply(ClusterApplierService.java:150)
  2> 	at org.elasticsearch.cluster.service.ClusterApplierService.runTask(ClusterApplierService.java:394)
  2> 	at org.elasticsearch.cluster.service.ClusterApplierService$UpdateTask.run(ClusterApplierService.java:155)
  2> 	at org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingRunnable.run(ThreadContext.java:569)
```

Relates to https://github.com/elastic/elasticsearch/issues/25335
PR to address the issue: https://github.com/elastic/elasticsearch/pull/29140

**Comment by DaveCTurner** [comment]
Thanks @vladimirdolzhenko, this does look like it'll be fixed by #29140 since `TransportNodesListShardStoreMetaData` is implicated:

```
  1> [2018-05-17T07:18:40,834][DEBUG][o.e.i.s.TransportNodesListShardStoreMetaData] [node_t0] /var/lib/jenkins/workspace/elastic+elasticsearch+6.2+matrix-java-periodic/ES_BUILD_JAVA/java9/ES_RUNTIME_JAVA/java10/nodes/virtual&&linux/server/build/testrun/integTest/J1/temp/org.elasticsearch.indices.recovery.IndexRecoveryIT_EA1D1070A4CA015-001/tempDir-006/data/nodes/0/indices/c-nzp85GT5y-dvCSSl9K-Q/0/_state/state-1.st: failed to read [state-], ignoring...
  1> java.nio.file.NoSuchFileException: /var/lib/jenkins/workspace/elastic+elasticsearch+6.2+matrix-java-periodic/ES_BUILD_JAVA/java9/ES_RUNTIME_JAVA/java10/nodes/virtual&&linux/server/build/testrun/integTest/J1/temp/org.elasticsearch.indices.recovery.IndexRecoveryIT_EA1D1070A4CA015-001/tempDir-006/data/nodes/0/indices/c-nzp85GT5y-dvCSSl9K-Q/0/_state/state-1.st
  1> 	at sun.nio.fs.UnixException.translateToIOException(UnixException.java:92) ~[?:?]
  1> 	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:111) ~[?:?]
  1> 	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:116) ~[?:?]
  1> 	at sun.nio.fs.UnixFileSystemProvider.newByteChannel(UnixFileSystemProvider.java:215) ~[?:?]
  1> 	at org.apache.lucene.mockfile.FilterFileSystemProvider.newByteChannel(FilterFileSystemProvider.java:212) ~[lucene-test-framework-7.2.1.jar:7.2.1 b2b6438b37073bee1fca40374e85bf91aa457c0b - ubuntu - 2018-01-10 00:48:44]
  1> 	at org.apache.lucene.mockfile.FilterFileSystemProvider.newByteChannel(FilterFileSystemProvider.java:212) ~[lucene-test-framework-7.2.1.jar:7.2.1 b2b6438b37073bee1fca40374e85bf91aa457c0b - ubuntu - 2018-01-10 00:48:44]
  1> 	at java.nio.file.Files.newByteChannel(Files.java:369) ~[?:?]
  1> 	at java.nio.file.Files.newByteChannel(Files.java:415) ~[?:?]
  1> 	at org.apache.lucene.store.SimpleFSDirectory.openInput(SimpleFSDirectory.java:77) ~[lucene-core-7.2.1.jar:7.2.1 b2b6438b37073bee1fca40374e85bf91aa457c0b - ubuntu - 2018-01-10 00:48:43]
  1> 	at org.elasticsearch.gateway.MetaDataStateFormat.read(MetaDataStateFormat.java:187) ~[main/:?]
  1> 	at org.elasticsearch.gateway.MetaDataStateFormat.loadLatestState(MetaDataStateFormat.java:322) [main/:?]
  1> 	at org.elasticsearch.index.shard.ShardPath.loadShardPath(ShardPath.java:122) [main/:?]
  1> 	at org.elasticsearch.indices.store.TransportNodesListShardStoreMetaData.listStoreMetaData(TransportNodesListShardStoreMetaData.java:142) [main/:?]
  1> 	at org.elasticsearch.indices.store.TransportNodesListShardStoreMetaData.nodeOperation(TransportNodesListShardStoreMetaData.java:109) [main/:?]
  1> 	at org.elasticsearch.indices.store.TransportNodesListShardStoreMetaData.nodeOperation(TransportNodesListShardStoreMetaData.java:61) [main/:?]
  1> 	at org.elasticsearch.action.support.nodes.TransportNodesAction.nodeOperation(TransportNodesAction.java:140) [main/:?]
  1> 	at org.elasticsearch.action.support.nodes.TransportNodesAction$NodeTransportHandler.messageReceived(TransportNodesAction.java:262) [main/:?]
  1> 	at org.elasticsearch.action.support.nodes.TransportNodesAction$NodeTransportHandler.messageReceived(TransportNodesAction.java:258) [main/:?]
  1> 	at org.elasticsearch.transport.AssertingTransportInterceptor$1.messageReceived(AssertingTransportInterceptor.java:78) [framework-6.2.5-SNAPSHOT.jar:?]
  1> 	at org.elasticsearch.transport.RequestHandlerRegistry.processMessageReceived(RequestHandlerRegistry.java:66) [main/:?]
  1> 	at org.elasticsearch.transport.TransportService$7.doRun(TransportService.java:656) [main/:?]
  1> 	at org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:672) [main/:?]
  1> 	at org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:37) [main/:?]
  1> 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1135) [?:?]
  1> 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) [?:?]
  1> 	at java.lang.Thread.run(Thread.java:844) [?:?]
```
